IMG/cpio/ventoy/hook/debian/pve-wa.sh

```
@@ -0,0 +1,20 @@
+#!/bin/sh
+
+script=$1
+tmpscript=${script}.tmp
+
+LINE=$(grep -n 'proc.*proc.*proc' $script | awk -F':' '{print $1}')
+
+sed -n "1,${LINE}p" $script >> $tmpscript
+
+echo "if grep -q 'rdinit=/vtoy/vtoy' /proc/cmdline; then"    >> $tmpscript
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)